nbgrader/tests/test_api.py

```diff
@@ -2,9 +2,12 @@
 
 from sqlalchemy import create_engine
 from sqlalchemy.orm import sessionmaker, scoped_session
+from nose.tools import assert_equal as _assert_equal
 
 from nbgrader import api
 
+assert_equal = lambda x, y: _assert_equal(x, y)
```

Comment:
Oh, yeah, this is not necessary. I added that in there because my linter was complaining that `assert_equal` needed three arguments (even though it doesn't), and the visual noise was really distracting. I can remove it now that I'm done.